api/envoy/config/filter/http/rbac/v2/rbac.proto

```diff
@@ -11,19 +11,30 @@ import "gogoproto/gogo.proto";
 // [#protodoc-title: RBAC]
 // Role-Based Access Control :ref:`configuration overview <config_http_filters_rbac>`.
 
+// RBAC filter config.
+// User sees the shadow rules as replacement to the enforced rules.
+// For example, policy A runs in prod in enforced mode, user wants
+// to test policy B; in the config, rules contains {A.mode=enforced},
+// shadow_rules contains {A.mode=shadow, B.mode=shadow}.
+// This will allow users to test policy replacement/deletion scenarios.
 message RBAC {
-  // Specify the RBAC rules to be applied globally
-  config.rbac.v2alpha.RBAC rules = 1 [(validate.rules).message.required = true];
+  // Specify the enforced RBAC rules to be applied globally.
+  // Filter's decision based on enforced rules has impact on user experience.
```

Comment:
I would replace these two sentences with: "If absent, no enforcing RBAC policy will be applied."

Reply:
done.